core/src/com/google/inject/internal/package-info.java

```diff
@@ -17,5 +17,4 @@
 /**
  * <i>Guice</i> (sounds like like "juice")
  */
-
-package com.google.inject.internal;
```

Comment:
This diff is hard to read.  The diff *looks* like it's deleting the package line.  But the file it's merging from has the package line (and so did the internal CL this was created from).  The package line is staying, right?